hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMOpenKeysDeleteRequest.java

```diff
@@ -106,15 +106,14 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       }
 
       omClientResponse = new OMOpenKeysDeleteResponse(omResponse.build(),
-          deletedOpenKeys, updatedBuckets, ozoneManager.isRatisEnabled());
+          deletedOpenKeys, updatedBuckets, ozoneManager.isRatisEnabled(),
+          getBucketLayout());
```

Comment:
With the proto update from #3226, the bucket type in the request will need to be used here, since the bucket may no longer exist at this point. I think that change will have more effects on this class as well.